src/app/globals.css

```
@@ -55,7 +55,7 @@ body {
 
 h1, h2, h3, h4, h5, h6 {
   font-family: var(--font-heading);
-  color: var(--color-manima-red); /* Updated to use theme color */
+  /* color: var(--color-manima-red); Updated to use theme color */
```

Stylelint (17.2.0)

[error] Expected empty line before comment (comment-empty-line-before)

(comment-empty-line-before)